projects/pulsar_lvds_adc/zed/system_project.tcl

```diff
@@ -0,0 +1,44 @@
+###############################################################################
+## Copyright (C) 2022-2024 Analog Devices, Inc. All rights reserved.
+### SPDX short identifier: ADIBSD
+###############################################################################
+
+# load script
+source ../../../scripts/adi_env.tcl
+source $ad_hdl_dir/projects/scripts/adi_project_xilinx.tcl
+source $ad_hdl_dir/projects/scripts/adi_board.tcl
+
+# RESOLUTION_16_18N - The resolution of the ADC
+# LEGEND: 18 BITS RESOLUTION AD7960 - 0
+#         16 BITS RESOLUTION AD7626 - 1
+
+set RESOLUTION_16_18N 0
```

Comment:
Use make parameters instead of this static aproach. You can use for reference AD4630, AD7616, AD7606. The project should build in a similar way as is described here

https://github.com/analogdevicesinc/hdl/blob/main/projects/ad4630_fmc/zed/README.md